db/migrate/20180124190951_create_documents.rb

```diff
@@ -1,14 +1,16 @@
 class CreateDocuments < ActiveRecord::Migration[5.1]
   def change
     create_table :documents do |t|
-      t.integer :customer_id
-      t.string :upload_id
-      t.string :upload_filename
-      t.string :upload_content_size
-      t.string :upload_content_type
-      t.string :doc_type
-      t.string :doc_number
-      t.date :doc_expire
+      t.string      :upload_id
+      t.string      :upload_filename
+      t.string      :upload_content_size
+      t.string      :upload_content_type
+      t.string      :doc_type
+      t.string      :doc_number
+
+      t.date        :doc_expire
+
+      t.references  :customer, null: false, index: true, foreign_key: true
```

Comment:
I always put my foreign keys at the beginning of the table, it actually helps natural join in SQL